runtime/services/catalog/migrator/sources/sources.go

```diff
@@ -301,12 +302,19 @@ func mergeFromParsedQuery(apiSource *runtimev1.Source) error {
 	if !ok {
 		return errors.New("unknown source")
 	}
-	if c == "local_file" {
+	switch c {
+	case "https":
 		return nil
+	case "local_file":
+		queryStr, err = rewriteLocalRelativePath(ast, repoRoot, strings.EqualFold(env["allow_host_access"], "true"))
+		if err != nil {
+			return err
+		}
+	default:
+		apiSource.Connector = c
+		props["path"] = p
```

Comment:
Maybe only do this for s3 and gcs specifically? And push to DuckDB in all unknown cases